code/game/objects/structures/inflatable.dm

```diff
@@ -439,3 +439,41 @@
 			update_icon()
 			to_chat(user,"<span class='notice'>You climb free of the shelter.</span>")
 
+/obj/structure/inflatable/shelter/MouseDropTo(atom/movable/O as mob|obj, mob/user as mob) //copy pasted from cryo code
+	if(!ismob(O))
+		return
+	if(O.loc == user || !isturf(O.loc) || !isturf(user.loc) || !user.Adjacent(O)) //no you can't pull things out of your ass
+		return
+	if(user.incapacitated() || user.lying) //are you cuffed, dying, lying, stunned or other
+		return
+	if(!Adjacent(user) || !user.Adjacent(src) || user.contents.Find(src)) // is the mob too far away from you, or are you too far away from the source
```

Comment:
user.contents.Find(src) shouldn't apply so you can remove that

Reply:
In theory I think it's for if you have a mob in your hands (ie, Ian, the pets etc).